share/functions/__fish_print_help.fish

```diff
@@ -11,7 +11,7 @@ function __fish_print_help --description "Print help message for the specified f
     end
 
     # Do nothing if the file does not exist
-    if not test -e "$__fish_data_dir/man/man1/$item.1" -o -e "$__fish_data_dir/man/man1/$item.1.gz"
+    if not path is -- $__fish_data_dir/man/man1/$item.1 $__fish_data_dir/man/man1/$item.1.gz; and not status get-file man/man1/$item.1 >/dev/null
```

Comment:
people may be surprised that we're stat-ing paths like "/man/man1/abbr.1",
if we really care we can add `set -q __fish_data_dir[1]`.
(AFAICT __fish_data_dir[1] is only set when turning off the embed feature)

Reply:
> people may be surprised that we're stat-ing paths like "/man/man1/abbr.1",

We aren't - the variable is an empty list, the entire argument is eliminated.